ocs_ci/deployment/baremetal.py

```diff
@@ -685,10 +686,16 @@ def clean_disk():
     """
     lvm_to_clean = []
     workers = get_nodes(node_type="worker")
+
     ocp_obj = ocp.OCP()
+    ocp_obj.new_project(project_name=constants.BM_DEBUG_NODE_NS)
```

Comment:
Where is the `constants.BM_DEBUG_NODE_NS` defined? (I don't see it in the [`constants.py`](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/constants.py) module and also it is not added in this PR.

Reply:
Updated